### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by hareesh114** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#247]
Which loss function did you use during training,  What is your model architecture.

**Comment by Rajamohanreddyai** [comment]
Hi, check this link
https://youtu.be/nZUxoHPFf4w

faster rcnn inception v2 architecture
